e2e/e2etests/test_migrate_connector_funds.go

```diff
@@ -0,0 +1,116 @@
+package e2etests
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/zeta-chain/node/e2e/runner"
+	"github.com/zeta-chain/node/e2e/utils"
+	observertypes "github.com/zeta-chain/node/x/observer/types"
+)
+
+// TestMigrateConnectorFunds tests the migration of funds from the old ZetaConnectorEth (V1) to the new ZetaConnectorNative (V2)
+func TestMigrateConnectorFunds(r *runner.E2ERunner, _ []string) {
+	r.Logger.Print("Migrating connector funds from V1 to V2")
+	ensureTxReceipt := func(tx *ethtypes.Transaction, failMessage string) {
+		receipt := utils.MustWaitForTxReceipt(r.Ctx, r.EVMClient, tx, r.Logger, r.ReceiptTimeout)
+		utils.RequireTxSuccessful(r, receipt, failMessage)
+	}
+
+	pauseConnectors(r, ensureTxReceipt)
+	defer unpauseConnectors(r, ensureTxReceipt)
+
+	updateTssAddress(r, r.EVMAddress(), ensureTxReceipt)
+	defer updateTssAddress(r, r.TSSAddress, ensureTxReceipt)
+
+	balanceTransferred := transferAllFunds(r, ensureTxReceipt)
+	verifyMigrationSuccess(r, balanceTransferred)
+}
+
+func updateTssAddress(
+	r *runner.E2ERunner,
+	tssAddress common.Address,
+	ensureTxReceipt func(*ethtypes.Transaction, string),
+) {
+	updateTssTx, err := r.ConnectorEth.UpdateTssAddress(r.EVMAuth, tssAddress)
+	require.NoError(r, err)
+	ensureTxReceipt(updateTssTx, "ZetaConnectorEth TSS address update failed")
+}
+
+func transferAllFunds(r *runner.E2ERunner, ensureTxReceipt func(*ethtypes.Transaction, string)) *big.Int {
```

Comment:
```suggestion
func transferAllFundsFromConnector(r *runner.E2ERunner, ensureTxReceipt func(*ethtypes.Transaction, string)) *big.Int {
```

Reply:
This is a private function for the file `test_migrate_connector_funds`. I think keeping the name is fine, to keep it concise